src/specklepy/objects/interfaces.py

```diff
@@ -63,32 +62,32 @@ def units(self, value: str | Units):
 
 @dataclass(kw_only=True)
 class IHasArea(metaclass=ABCMeta):
-
     _area: float = field(init=False, repr=False)
 
     @property
+    @abstractmethod
```

Comment:
if its an abstract method, you should not provide an implementation

Reply:
removed the implementation, we don't need that since the classes have their unique area and volume implementations.